DeanerySystem/DeanerySystem.DataAccess/Configurations/ClassNumberTimeConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+﻿using DeanerySystem.DataAccess.Entities;
+using DeanerySystem.DataAccess.Utilities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace DeanerySystem.DataAccess.Configurations
+{
+    class ClassNumberTimeConfiguration : DbEntityConfiguration<ClassNumberTime>
+    {
+        public override void Configure(EntityTypeBuilder<ClassNumberTime> entity)
+        {
+			entity.ToTable("ClassNumberTimes");
+			entity.HasKey(t => t.Id);
+			entity.Property(t => t.Id).ValueGeneratedOnAdd();
+
+            entity.HasOne(t => t.TimeTable)
```

Comment:
I know that this has been written like that from the start. But I think it would be good to:
- rename this table to something some meaningful
- have a many-to-many relationship with `TimeTable`

Reply:
Can you please provide more details?